autorag/indexer/process/azure/paragraph.py

```diff
@@ -0,0 +1,111 @@
+from llama_index import Document
+from llama_index.schema import TextNode
+from llama_index.node_parser import SentenceSplitter
+
+# Define roles to be excluded
+DEFAULT_EXCLUDED_ROLES: list[str] = ["pageHeader", "pageNumber"]
+
+# Define contents to be excluded
+DEFAULT_EXCLUDED_CONTENTS: list[str] = ["Contains Nonbinding Recommendations"]
+
+DEFAULT_FILE_TYPE: str = "Guidance"
+
+
+class AzureParagraphProcessor:
+    """
+    Process azure paragraphs list
+
+    :param azure_paragraphs_list: The list of paragraphs returned from Azure.
+    :param file_name: The name of the file.
```

Comment:
Minor: docstring for `sentence_splitter_args`

Reply:
Good point and much appreciated, I've made the necessary changes.